gazelle/MODULE.bazel

```diff
@@ -17,14 +17,25 @@ local_path_override(
 
 go_deps = use_extension("@bazel_gazelle//:extensions.bzl", "go_deps")
 go_deps.from_file(go_mod = "//:go.mod")
+go_deps.module_override(
+    patches = ["//patches:go-tree-sitter-cc_library.patch"],
+    path = "github.com/tree-sitter/go-tree-sitter",
+    patch_strip = 1,
+)
+go_deps.module_override(
+    path = "github.com/tree-sitter/tree-sitter-python",
```

Comment:
One alternative to patching the `tree-sitter-*` libraries (with the `bindings/go/bindings.go` file) is generating a single BUILD ourselves like what I've done in the aspect-cli, see [the http fetch+BUILD](https://github.com/aspect-build/aspect-cli/blob/d89f6e83dc94ccd926d718a0506fd944f5826802/gazelle/common/treesitter/grammars/grammars.bzl#L7-L23) and [associated go](https://github.com/aspect-build/aspect-cli/blob/d89f6e83dc94ccd926d718a0506fd944f5826802/gazelle/common/treesitter/grammars/golang/binding.go).

That's probably nicer then a patch now that I look at it again...?

Reply:
Using go modules works better when mixing go modules across multiple modules though, such as multiple gazelle languages from different modules. But that requires the patches :/

Reply:
Making the BUILD file ourselves is basically what the dougthor42/go-tree-sitter does.

Does manually setting `http_archive.build_file_content` work with both WORKSPACE and bzlmod? If so that might be a way forward. I agree it's cleaner than the patches.

Reply:
This is only for the `tree-sitter-{lang}/bindings/go/bindings.go` files, not the main `go-tree-sitter` go module that has the bigger problem.